components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -442,6 +445,42 @@ export function CreateWorkspacePage() {
     );
 }
 
+function LocalSSHWarning(props: { className: string }) {
+    const isLocalSSHProxyEnabled = useFeatureFlag("gitpod_desktop_use_local_ssh_proxy");
+
+    if (!isLocalSSHProxyEnabled) {
+        return null;
+    }
+
+    return (
+        <div className={props.className}>
+            <Alert light className="bg-gray-100 dark:bg-gray-800" type="info">
```

Comment:
**issue(non-blocking)**: The placement of the alert still seems inappropriate as described in https://github.com/gitpod-io/gitpod/pull/18112#discussion_r1253042041, but since there are no strong objections and this is a temporary experiment let's keep it as is. 🤝